accepted/system-range/system-range.md

```diff
@@ -32,25 +31,33 @@ so that we can look at source code as it's likely written:
 
 ```csharp
 var fiveToTen = 5..11; // Equivalent to Range.Between(5, 11)
+
+// You can also construct ranges with index and length:
+var fiveToTen = 5.:6; // Equivalent to new Range.Between(5, 6)
 ```
 
-### Open ended ranges
+### Unbounded ranges
 
 ```csharp
-// Ranges can be open ended:
+// Ranges can be unbounded:
 
 var fiveToEnd = 5..;  // Equivalent to Range.From(5) i.e. missing upper bound
 var startToTen = ..1; // Equivalent to Range.To(11) i.e. missing lower bound
 var everything = ..;  // Equivalent to Range.All i.e. missing upper and lower bound
 
-// Asking for Index, Length or enumerating an open range will throw
+// When the lower bound is omitted, it's interpreted to be zero.
+//
+// When the upper bound is omitted, it's interpreted to be the length of the
+// receiving collection.
```

Comment:
I think we should specify this is only true when methods, as an implementation detail, call the Bind method. It's possible for methods to interpret unbounded ends any way they choose. Although most will simply use Bind.